subset/password/brute_test/src/main/java/PackageManager.java

```diff
@@ -0,0 +1,119 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class PackageManager {
+
+  String host;
+  int connectionPort;
+  String macAddress;
+  SshSocket sshSocket = null;
+  TelnetSocket telnetSocket = null;
+  String protocol;
+  Map<String, String> macDevices = new HashMap<String, String>();
+  boolean debug = false;
+
+  public void readFile() {
+
+		 try {
+              System.out.println("Attempting to read web file...");
+
+              InputStream url = new URL("https://svn.nmap.org/nmap/nmap-mac-prefixes").openStream();
+      //      InputStream url = this.getClass().getResourceAsStream("resources/macList.txt");
+              StringBuilder resultStringBuilder = new StringBuilder();
+	          try (BufferedReader br= new BufferedReader(new InputStreamReader(url))){
+	              String line;
+	              while ((line = br.readLine()) != null) {
+	                  resultStringBuilder.append(line).append("\n");
+	                 // System.out.println(line);
+	                  String[] words = new String[2];
+	                  String macAddress;
+	                  String manufacturer;
+	                  if(line.length() > 5) {

```

Comment:
random constants again. Please find a more meaningful way to represent this.  why 5?  CONSTANT.length()?